src/robusta/cli/slack_verification.py

```diff
@@ -0,0 +1,70 @@
+import typer
+from slack_sdk import WebClient
+from slack_sdk.errors import SlackApiError
+
+SLACK_WELCOME_MESSAGE_TITLE = ":large_green_circle: INFO - Welcome to Robustas"
+SLACK_WELCOME_MESSAGE_HEADER_WITH_CLUSTER_NAME = "You have signed up for alerts for cluster: '{}'."
```

Comment:
I think 'alerts' is a little undermining.
I would try something more impressive.
signed for insights, monitoring, or something